Type error when tweened object is a Three.Vector3

## Expected Behavior

new tween instance without a type error.

## Current Behavior

error TS2345 is It happens, ``new TWEEN.Tween(cube.position)``

## Possible Solution

TWEEN.Tween(cube.position as any), type errors are not occur.
It is not happen in tween.js 18.5.0 + @types/tween.js 18.5.1

## Steps to Reproduce

Reproduction Repository
https://github.com/MasatoMakino/behavior-of-new-Tween.tween

1. clone repository
2. ``npm run tsc``
3. error TS2345

error log
```
node_modules/three/src/core/BufferAttribute.d.ts:164:10 - error TS2304: Cannot find name 'Iterable'.

164     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:174:10 - error TS2304: Cannot find name 'Iterable'.

174     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:184:10 - error TS2304: Cannot find name 'Iterable'.

184     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:194:10 - error TS2304: Cannot find name 'Iterable'.

194     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:204:10 - error TS2304: Cannot find name 'Iterable'.

204     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:214:10 - error TS2304: Cannot find name 'Iterable'.

214     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:224:10 - error TS2304: Cannot find name 'Iterable'.

224     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:234:10 - error TS2304: Cannot find name 'Iterable'.

234     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

node_modules/three/src/core/BufferAttribute.d.ts:244:10 - error TS2304: Cannot find name 'Iterable'.

244     array: Iterable<number> | ArrayLike<number> | ArrayBuffer | number,
               ~~~~~~~~

src/TestNewTween.ts:7:31 - error TS2345: Argument of type 'Vector3' is not assignable to parameter of type 'Record<string, unknown>'.
  Index signature is missing in type 'Vector3'.

7 const tween = new TWEEN.Tween(cube.position).to({x: 100, y: 100, z: 100}, 10000); // NG
                                ~~~~~~~~~~~~~
```

## Context (Environment)

Environment
```package.json
"@tweenjs/tween.js": "^18.6.0",
"three": "^0.118.1",
"typescript": "^3.9.5"
```

I was trying to animate the Mesh object in three.js with tween.js.

I referred to the code in the tween.js user guide, but I get a type error.
[tween.js user guide](https://github.com/tweenjs/tween.js/blob/master/docs/user_guide.md#tweenjs-user-guide)

I couldn't determine if the problem was due to my typescript configuration or the type definition file in 18.6.0. If anyone has any information, please let me know.

Thank you.

--- a/src/Tween.ts
+++ b/src/Tween.ts
@@ -21,7 +21,7 @@ export class Tween<T extends UnknownProps> {
 	private _isPaused = false
 	private _pauseStart = 0
 	private _valuesStart: UnknownProps = {}
-	private _valuesEnd: UnknownProps = {}
+	private _valuesEnd: Record<string, number | string> = {}
 	private _valuesStartRepeat: UnknownProps = {}
 	private _duration = 1000
 	private _initialRepeat = 0
@@ -477,11 +477,10 @@ export class Tween<T extends UnknownProps> {
 
 	private _swapEndStartRepeatValues(property: string): void {
 		const tmp = this._valuesStartRepeat[property]
+		const endValue = this._valuesEnd[property]
 
-		if (typeof this._valuesEnd[property] === 'string') {
-			// eslint-disable-next-line
-			// @ts-ignore FIXME?
-			this._valuesStartRepeat[property] = this._valuesStartRepeat[property] + parseFloat(this._valuesEnd[property])
+		if (typeof endValue === 'string') {
+			this._valuesStartRepeat[property] = this._valuesStartRepeat[property] + parseFloat(endValue)
 		} else {
 			this._valuesStartRepeat[property] = this._valuesEnd[property]
 		}
@@ -490,6 +489,7 @@ export class Tween<T extends UnknownProps> {
 	}
 }
 
-export type UnknownProps = Record<string, unknown>
+// eslint-disable-next-line
+export type UnknownProps = Record<string, any>
 
 export default Tween

